### PR TITLE
Added rdctl command reference doc

### PR DESCRIPTION
**Pull request by gunamata**
Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

### CODE REVIEW

File: docs/references/rdctl-command-reference.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: rdctl Command Reference
+---
+
+`rdctl` is a command-line tool, included in Rancher Desktop that enables command-line access to the application features. `rdctl` is developed to help users with tasks such as scripting (for automation, CI/CD), troubleshooting, remote management etc. The current version of `rdctl` supports below commands, and support for more commands would be added in the upcoming releases.
```
Comment (ericpromislow) [inline docs/references/rdctl-command-reference.md:5]: `rdctl` is a command-line tool, included in Rancher Desktop that enables command-line access to GUI features. `rdctl` is developed to help users with tasks such as scripting (for automation, CI/CD), troubleshooting, remote management, etc. The current version of `rdctl` supports the below commands (with support for more commands to be added in upcoming releases):

File: docs/references/rdctl-command-reference.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+***Note:  Rancher Desktop app should be running on your machine to use `rdctl` commands ***
```
Comment (ericpromislow) [inline docs/references/rdctl-command-reference.md:7]: app must be running

-- It won't work otherwise.

**Review by ericpromislow — CHANGES_REQUESTED** [review]
The rest looks good.

File: docs/references/rdctl-command-reference.md (hunk 1 above)
Comment (gunamata) [reply docs/references/rdctl-command-reference.md]: Modified per your suggestion.

File: docs/references/rdctl-command-reference.md (hunk 2 above)
Comment (gunamata) [reply docs/references/rdctl-command-reference.md]: Modified per your suggestion.

File: docs/references/rdctl-command-reference.md (hunk 1 above)
Comment (jandubois) [reply docs/references/rdctl-command-reference.md]: I think it should mention that the current version is experimental and all subcommands names, their arguments, and their output are still subject to change.

**Review by ericpromislow — APPROVED** [review]
LGTM

File: docs/references/rdctl-command-reference.md (hunk 1 above)
Comment (gunamata) [reply docs/references/rdctl-command-reference.md]: @jandubois , I have added a warning note that it's experimental....

**Pull request merged**